.github/workflows/copyright-validate.yaml

```diff
@@ -1,10 +1,15 @@
-name: Check Copyright Headers
+name: Add Copyright Headers
 on: pull_request
 jobs:
   copywrite:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Is this needed to be able to commit?

Reply:
[More infor about GH action permissions](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#overview).

This allows the action, use the `GITHUB_TOKEN` with write permissions within the content of the repo, so it can change files and commit them.

But I based the decission of adding it as per documentation, I did not perform a run to test if we can write the files, generate a commit and pushed without this rule.